examples/s3-replication/README.md

```diff
@@ -29,16 +29,16 @@ Note that this example may create resources which cost money. Run `terraform des
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 3.50 |
-| <a name="provider_aws.replica"></a> [aws.replica](#provider\_aws.replica) | >= 3.50 |
-| <a name="provider_random"></a> [random](#provider\_random) | >= 2.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 3.64.1 |
```

Comment:
Please update `versions.tf` in the example also, because it won't work with older requirements (3.50)

Reply:
yes thanks for spotting this I got overwhelmed with the push ;)